releases/release-1.31/release-notes/maps/pr-124503-map.yaml

```
@@ -0,0 +1,6 @@
+pr: 124503
+releasenote:
+  text: 'kubeadm: during the preflight check "CreateJob" of "kubeadm upgrade", check
+    if there are no nodes where a Pod can schedule. If there are none, show a warning
+    and skip this preflight check. This can happen in single node clusters where the
+    only node was drained.'
```

```suggestion
  text: 'kubeadm: Added support during the preflight check "CreateJob" of "kubeadm upgrade" to check if there are no nodes where a Pod can be scheduled. If there are none, show a warning and skip this preflight check. This can happen in single node clusters where the only node was drained.'
```

https://github.com/kubernetes/sig-release/pull/2542#discussion_r1653319211